src/ucs/stats/serialization.c

```diff
@@ -283,11 +303,18 @@ ucs_stats_serialize_binary(FILE *stream, ucs_stats_node_t *root,
     for (elem = sglib_hashed_ucs_stats_clsid_t_it_init(&it, cls_hash);
          elem != NULL; elem = sglib_hashed_ucs_stats_clsid_t_it_next(&it))
     {
+        ucs_stats_filter_node_t *filter_node;
+        unsigned num_counters_filtered;
+
         cls = elem->cls;
         ucs_stats_write_str(cls->name, stream);
-        FWRITE_ONE(&cls->num_counters, stream);
-        for (counter = 0; counter < cls->num_counters; ++counter) {
-            ucs_stats_write_str(cls->counter_names[counter], stream);
+
+        filter_node = elem->filter_node;
```

Comment:
minor: please align by `=` with the line below

Reply:
Do you mean like this:
        filter_node                   = elem->filter_node;
        num_counters_filtered = __builtin_popcount(elem->filter_node->counters_bitmask);

?

Fixed.